### ISSUE TITLE
parcel: not found with yarn (pnp)

### ISSUE THREAD

**Issue by dhoko** [parcel-bundler/parcel#2635]
# 🐛 bug report

I can't use parcel (_npm scripts_) if I have yarn with pnp active.

## 🤔 Expected Behavior

```
$ yarn start
yarn run v1.13.0
$ parcel index.html
Server running at http://localhost:1234 
✨  Built in 458ms.
```

As it is if pnp is disable.

## 😯 Current Behavior

`/bin/sh: 1: parcel: not found `

![screenshot from 2019-02-12 15-04-59](https://user-images.githubusercontent.com/713283/52640781-98ec9a80-2ed7-11e9-9ab1-4f91a2bc5ad7.png)

## 🔦 Context

I wanted to try using yarn + pnp

## 💻 Code Sample

https://github.com/dhoko/demo-parcel

## 🌍 Your Environment

| Software         | Version(s) |
| ---------------- | ---------- |
| nvm              | 0.26.1
| Node             | v11.6.0 
| Yarn             | 1.13.0
| Operating System | Ubuntu 18.04.1 LTS

**Comment by mischnic** [comment]
I'm quite sure this is the same issue as https://github.com/yarnpkg/yarn/issues/6917

**Comment by quasicomputational** [comment]
This is definitely caused proximally by that Yarn bug, but the reason Parcel is ending up unplugged is because it has a postinstall script, which exists for non-technical purposes. If you install it with `--ignore-scripts`, then you can run the `parcel` command.

But it very likely won't actually work, because Parcel doesn't know about PnP resolution yet. I had a brief look at `Resolver.js`, but it's not an entirely trivial change and would need tests, so, yeah.

**Comment by frunkad** [comment]
Anytime when this will be fixed?

**Comment by DeMoorJasper** [comment]
@frunkad it'll be fixed in Parcel 2, see #3582

**Comment by mischnic** [comment]
Closing as a non-parcel-bug
